frog-utils/src/index.js

```diff
@@ -47,8 +47,18 @@ export const splitAt = (i: number, xs: Array<any>): Array<Array<any>> => {
   return [first, second];
 };
 
-export const shuffleList = (xs: Array<any>): Array<any> =>
-  xs.slice(0).sort(() => 0.5 - Math.random());
+export const shuffleList = (xs: Array<any>): Array<any> => {
```

Comment:
Would it make sense to replace this with lodash.shuffle? https://lodash.com/docs/4.17.4#shuffle